SPEC.md

```diff
@@ -155,6 +155,37 @@ When persisting an output artifact to a file system, if the build tool has addit
 it should persist that metadata to a subdirectory of the directory to which the output artifact is being written of the form: ```.bom/metadata/${tool}/```.  Filenaming and subdirectory structure below that point is at the discretion of the build tool.
 ## Annex A
 
+## Annex B
+
+### Build tool selection of gitoid type
+
+GITOID_BLOB_TYPE should be set in order of precedence by:
+1. A non-empty environment variable named GITOID_BLOB_TYPE.
```

Comment:
which should taker higher precedence?

I think the tool-specific option should take higher precedence. for example, the gcc -fgitoid-type=0,2 option should take effect even if an environmental variable GITOID_BLOB_TYPE exists. When gcc command line option is missing and an environmental variable GITOID_BLOB_TYPE exists. then the env-var takes effect. When both are missing, then the default gitoid like sha1 will be used.

Reply:
Do we anticipate using a non-blob GITOID type?

Reply:
> which should taker higher precedence?
It was discussed earlier that typically (including languages such as 'Go'), the precedence is generally given to environment variables.  So environment variable overrides the command line option, if both are provided.

Reply:
> which should taker higher precedence?
> It was discussed earlier that typically (including languages such as 'Go'), the precedence is generally given to environment variables. So environment variable overrides the command line option, if both are provided.

OK, then it is fine. thanks for the explanation.